src/components/Header.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { ErrorMessages } from '../types/ErrorMessage';
+
+type Props = {
+  onToddoAdd: (todoTitle: string) => Promise<void>;
+  onToggleChange: (todo: Todo) => void;
+  todos: Todo[],
+  inputRef: React.RefObject<HTMLInputElement>;
+  setTempTodo: (value: Todo | null) => void;
+  isDisabled: boolean;
+  setIsDisabled: (value: boolean) => void;
+  setErrorMessage: (value: string) => void;
+  todoTitle: string;
+  setTodoTitle: (value: string) => void;
+};
+
+const USER_ID = 11516;
```

Comment:
I see this constant created in the API too. It must be only one `USER_ID`, move it constants and use it